### PR TITLE
Update WritableStream tests

### PR DESCRIPTION
**Pull request by tyoshino**
Spec change: https://github.com/whatwg/streams/pull/655

### CODE REVIEW

**Comment by tyoshino** [comment]
The spec side change is also still under review.

**Comment by tyoshino** [comment]
Rebased.

The last commit before rebase was: a81b8b3b18619b270ab3b0df064c6a60e07fca55

**Comment by tyoshino** [comment]
Rebased again onto fb0caa1eb036998eb2450d9c63dc5450a6f75578 to sync with the point specified for the submodule in the streams repo.

File: streams/writable-streams/write.js
Comment (domenic) [inline streams/writable-streams/write.js]: Since both of these have `error.name === 'Error'` the `promise_rejects` tests here are no good. Let's stick to the `error1` + `error2` pattern.

**Review by domenic — APPROVED** [review]
LGTM after the one passedError/thrownError thing is fixed.

**Comment by tyoshino** [comment]
Rebased and squashed. Old commit is f5c2f755bce917563d30c31be91838f28736a578.

File: streams/writable-streams/write.js
Comment (tyoshino) [inline streams/writable-streams/write.js]: Thanks. Removed passedError and thrownError from all the tests in this file and switched to use error1/error2 in a48c302e5a6af51f80c0a98439379af427411444

**Comment by tyoshino** [comment]
Squashed and updated the commit message. The last commit before this was a48c302e5a6af51f80c0a98439379af427411444.

**Comment by tyoshino** [comment]
Hi Domenic, it looks I need write access to commit. Can I get it or could you please merge this?

**Comment by domenic** [comment]
@sideshowbarker can you help give @tyoshino write access for the future? And @ricea while we're at it.

**Pull request merged**
